src/pkg/dir.traversal.targets

```diff
@@ -1,5 +1,5 @@
 <Project>
-  <Import Project="$(RepositoryEngineeringDir)dir.traversal.targets" />
+  <Import Project="..\dir.traversal.targets" />
```

Comment:
Why not use the `LibrariesProjectRoot` property?

Reply:
Relative paths are usually preferred.